src/server/readpool/context.rs

```diff
@@ -11,9 +11,121 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use std::fmt;
+use std::mem;
+use prometheus::local::{LocalCounterVec, LocalHistogramTimer, LocalHistogramVec};
+
 use util::futurepool;
+use util::worker;
+use util::collections::HashMap;
+use pd;
+use storage;
+
+use super::metrics::*;
+
+pub struct Context {
+    pd_sender: Option<worker::FutureScheduler<pd::PdTask>>,
+
+    command_duration: LocalHistogramVec,
```

Comment:
+1

Reply:
Would read operator in storage and coprocessor share the same metrics later? If yes, maybe we could tolerate it temporarily since it would be removed later?